src/MercadoPago/Exceptions/MPApiException.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+namespace MercadoPago\Exceptions;
+
+use Exception;
+use MercadoPago\Net\MPResponse;
+
+/** MPApiException class. */
+class MPApiException extends Exception
+{
+    private $status_code;
+
+    private $api_response;
+
+    /**
+     * MPApiException constructor.
+     * @param string $message exception message.
+     * @param \MercadoPago\Net\MPResponse $response response from the request.
+     */
+    public function __construct(string $message, MPResponse $response)
```

Comment:
@rhames07 chegou a explorar a opção de usar constructor property promotion do php8 ? Aqui tem um exemplo: https://www.tutorialspoint.com/constructor-property-promotion-in-php-8 . Se fosse possível, diminuiria a quantidade de linhas da classe.